examples/vue.test.js

```diff
@@ -2600,6 +2600,63 @@ describe('Vue', function() {
       // acquit:ignore:end
     });
   });
+  it('vue-websocket', async function() {
+    // acquit:ignore:start
+    this.timeout(10000);
+    const browser = await puppeteer.launch({ headless: false });
+    const page = await browser.newPage();
+    const fn = function() {
+    // acquit:ignore:end
+      const app = new Vue({
+        data: function(){
+          return{
+          connection: null,
+          time: null
+          }
+        },
+        errorCaptured: function(err) {
+          console.log('Caught top-level error', err.message);
+          return false;
+        },
+        methods:{
+          sendMessage: function(message){
+            console.log(this.connection);
+            this.connection.send(message);
+          },
+        },
+        template: `
+          <div>
+            <h1>Hello</h1>
+            <h2>{{time}}</h2>
+              <button v-on:click="sendMessage('hello')">
```

Comment:
Let's get rid of the whole `sendMessage()` method, it isn't necessary if you implement the server changes I mentioned.